systemtest/src/main/java/io/strimzi/systemtest/resources/operator/SetupClusterOperator.java

```diff
@@ -462,6 +465,63 @@ public void prepareEnvForOperator(ExtensionContext extensionContext, String clie
         prepareEnvForOperator(extensionContext, clientNamespace, Collections.singletonList(clientNamespace));
     }
 
+    private String changeLeaseNameInResourceIfNeeded(String yamlPath) {
+        final EnvVar leaseEnvVar = extraEnvVars.stream().filter(envVar -> envVar.getName().equals("STRIMZI_LEADER_ELECTION_LEASE_NAME")).findFirst().orElse(null);
+        if (leaseEnvVar != null && yamlPath.contains("022-")) {
+            try {
+                File tmpFile = File.createTempFile(yamlPath.replace(Constants.STRIMZI_DEPLOYMENT_NAME + ".yaml", leaseEnvVar.getValue()), "yaml");
+                String tmpFileContent = "";
+                final String resourceName = leaseEnvVar.getValue() + "-leader-election";
+                final String resourceType = yamlPath.replace(TestUtils.USER_PATH + "/../packaging/install/cluster-operator/", "").split("-")[1];
```

Comment:
don't we have `packaging/install...` in some constant?

Reply:
Nope, I search through the module and I didn't find anything
We are usually working with `TestUtils.USER_PATH + "...path"`

I think I can add it to `Constants` class and changed it everywhere, but I think that's something for different PR